kbs/config/kubernetes/custom_pccs/kustomization.yaml

```diff
@@ -3,15 +3,15 @@ kind: Kustomization
 namespace: coco-tenant
 
 resources:
-- ../nodeport/x86_64
+  - ../nodeport/
 
 patches:
-- path: set_custom_pccs.yaml
-  target:
-    kind: Deployment
-    name: kbs
+  - path: set_custom_pccs.yaml
+    target:
+      kind: Deployment
+      name: kbs
 
 configMapGenerator:
-- files:
-  - sgx_default_qcnl.conf
-  name: dcap-attestation-conf
+  - files:
+      - sgx_default_qcnl.conf
+    name: dcap-attestation-conf
```

Comment:
do you know what triggers these whitespaces here?

Reply:
Argh - sorry the linter in my editor auto-formats the yamls, which I missed on the latest changes